app/presenters/vendor_api/application_presenter.rb

```diff
@@ -112,6 +113,10 @@ def safeguarding_issues_details_url
       application_form.has_safeguarding_issues_to_declare? ? provider_interface_application_choice_url(application_choice, anchor: 'criminal-convictions-and-professional-misconduct') : nil
     end
 
+    def anonymised
+      application_form.candidate.email_address.include? 'deleted-application-'
```

Comment:
sorry to be annoying, but i think it'd be good to have this logic encapsulated in a service class, such that the presenter is just asking `IsApplicationAnonymised.new(application_form).call`

Reply:
🆗